src/test/test/controller.test.ts

```diff
@@ -510,6 +530,27 @@ describe("controller", () => {
     });
   });
 
+  describe("shouldQuickPickOpenFromSelection", () => {
+    it("1: should return false if quickPick is not initialized", () => {
+      setups.shouldQuickPickOpenFromSelection1();
+
+      assert.equal(controller.shouldQuickPickOpenFromSelection(), false);
+    });
+
+    it("2: should return false if fetchShouldQuickPickOpenFromSelection method returns false", () => {
+      setups.shouldQuickPickOpenFromSelection2();
+
+      assert.equal(controller.shouldQuickPickOpenFromSelection(), false);
+    });
+
+    it(`3: should return true if quickPick is already initialized,
+      fetchShouldQuickPickOpenFromSelection methods return true`, () => {
+      setups.shouldQuickPickOpenFromSelection3();
+
+      assert.equal(controller.shouldQuickPickOpenFromSelection(), true);
+    });
+  });
+
```

Comment:
```suggestion
    it("3: should return false if editor doesn't exist", () => {
      const editor = setups.shouldSearchSelection3();
      assert.equal(controller.shouldSearchSelection(editor), false);
    });

    it(`4: should return true if quickPick is initialized,
      fetchShouldSearchSelection returns false and editor exists`, () => {
      const editor = setups.shouldSearchSelection4();
      assert.equal(controller.shouldSearchSelection(editor), true);
    });
  });

```

Add one more test to verify if active editor exists.